core/frontend/src/tile/RealityTileLoader.ts

```diff
@@ -154,12 +154,30 @@ export abstract class RealityTileLoader {
 
     const center = location.multiplyPoint3d(tile.center, scratchTileCenterWorld);
     let minDistance = 1.0;
+
+    const now = Date.now();
+    const wheelEventRelevanceTimeout = 1000; // Wheel events older than this value will not be considered
+
     for (const viewport of viewports) {
       const npc = viewport.worldToNpc(center, scratchTileCenterView);
 
+      let focusPoint = new Point2d(0.5, 0.5);
+
+      if (viewport instanceof ScreenViewport) {
+        // Try to get a better target point from the last zoom target
+        const {lastWheelEvent} = IModelApp.toolAdmin.currentInputState;
```

Comment:
check if `currentInputState.viewport === viewport`, the zoom point is otherwise not relevant.